README.md

```diff
@@ -271,3349 +318,4157 @@ style guide.
 
-  ```Ruby
-  # bad
-  begin
-    puts val
-    val += 1
-  end while val < 0
+    ```ruby
+    # bad - easier to move/add/remove parameters, but still not preferred
+    some_method(
+      size,
+      count,
+      color,
+    )
 
-  # good
-  loop do
-    puts val
-    val += 1
-    break unless val < 0
-  end
-  ```
+    # bad
+    some_method(size, count, color, )
 
-* <a name="no-dsl-parens"></a>
-  Omit parentheses around parameters for methods that are part of an internal
-  DSL (e.g. Rake, Rails, RSpec), methods that have "keyword" status in Ruby
-  (e.g. `attr_reader`, `puts`) and attribute access methods. Use parentheses
-  around the arguments of all other method invocations.
-<sup>[[link](#no-dsl-parens)]</sup>
+    # good
+    some_method(size, count, color)
+    ```
 
-  ```Ruby
-  class Person
-    attr_reader :name, :age
+  * <a name="spaces-around-equals"></a>
+    Use spaces around the `=` operator when assigning default values to method
+    parameters:
+    <sup>[[link](#spaces-around-equals)]</sup>
 
-    # omitted
-  end
+    ```ruby
+    # bad
+    def some_method(arg1=:default, arg2=nil, arg3=[])
+      # do something...
+    end
 
-  temperance = Person.new('Temperance', 30)
-  temperance.name
+    # good
+    def some_method(arg1 = :default, arg2 = nil, arg3 = [])
+      # do something...
+    end
+    ```
 
-  puts temperance.age
+    While several Ruby books suggest the first style, the second is much more
+    prominent in practice (and arguably a bit more readable).
 
-  x = Math.sin(y)
-  array.delete(e)
+  * <a name="no-trailing-backslash"></a>
+    Avoid line continuation `\` where not required. In practice, avoid using
+    line continuations for anything but string concatenation.
+    <sup>[[link](#no-trailing-backslash)]</sup>
 
-  bowling.score.should == 0
-  ```
+    ```ruby
+    # bad
+    result = 1 - \
+             2
 
-* <a name="no-braces-opts-hash"></a>
-  Omit the outer braces around an implicit options hash.
-<sup>[[link](#no-braces-opts-hash)]</sup>
+    # good (but still ugly as hell)
+    result = 1 \
+             - 2
 
-  ```Ruby
-  # bad
-  user.set({ name: 'John', age: 45, permissions: { read: true } })
+    long_string = 'First part of the long string' \
+                  ' and second part of the long string'
+    ```
 
-  # good
-  user.set(name: 'John', age: 45, permissions: { read: true })
-  ```
+  * <a name="consistent-multi-line-chains"></a>
+    Adopt a consistent multi-line method chaining style. There are two popular
+    styles in the Ruby community, both of which are considered
+    good&mdash;leading `.` (Option A) and trailing `.` (Option B).
+    <sup>[[link](#consistent-multi-line-chains)]</sup>
 
-* <a name="no-dsl-decorating"></a>
-  Omit both the outer braces and parentheses for methods that are part of an
-  internal DSL.
-<sup>[[link](#no-dsl-decorating)]</sup>
+      * **(Option A)** When continuing a chained method invocation on
```

Comment:
Ro mentioned that option B is strongly preferred here for ability to copy and paste code- I think that makes sense and we can go with that decision unless anyone has a strong argument otherwise.

Reply:
:+1: trailing dots (option B) since it's what we've been doing

Reply:
I think option A breaks the ability to send code from vim to the Rails console via Tmux.

Reply:
I went ahead and updated this to be Option B here.

Reply:
Since this is a public repo I can chime in here to tell you all that Option A is the one true faith.  :trollface: 

FROM BEYOND THE GRAVE

Reply:
I hate trailing dots

Reply:
Hi @stephenprater ! 👋 

@mpeychich I find Option A to be more readable as well but I'm willing to forgo my opinion so our vim devs can utilize their tools for this. Let me know if you think we should bring this up further in person.